src/nft/pages/collection/index.tsx

```diff
@@ -1,5 +1,35 @@
-export const Collection = () => {
-  return <div>Collection Page</div>
+import { useWindowSize } from 'hooks/useWindowSize'
+import { GenieCollection } from 'nft/types'
+import { useEffect, useMemo } from 'react'
+import { useQuery } from 'react-query'
+import { useLocation, useParams } from 'react-router-dom'
+
+import { useIsMobile } from '../../hooks/useIsMobile'
+import { CollectionStatsFetcher } from '../../queries'
+import { CollectionDesktop } from './CollectionDesktop'
+import { CollectionMobile } from './CollectionMobile'
+
+const Collection = () => {
+  const { contractAddress } = useParams()
+
+  const isMobile = useIsMobile((state) => state.isMobile)
```

Comment:
You can elide L15-16 into one line: `const { isMobile, setMobileWidth } = useIsMobile()`